aten/src/ATen/cpu/vec256/functional.h

```diff
@@ -13,9 +13,10 @@ inline scalar_t vec_reduce_all(
   scalar_t acc_arr[Vec::size()];
   acc_vec.store(acc_arr);
   for (int64_t i = 1; i < size; i++) {
-    scalar_t acc_arr_next[Vec::size()];
+    std::array<scalar_t, Vec::size()> acc_arr_next;
+    acc_arr_next.fill(0);
```

Comment:
Those two should be identical, aren't they?
```suggestion
    std::array<scalar_t, Vec::size()> acc_arr_next = {0};
```